web/models/EmpresaDTO.php

```diff
@@ -77,7 +77,7 @@ public static function listar($termo = '')
         $sql = "SELECT * FROM empresa ";
 
         if (!empty($termo)) {
-            $sql .= "WHERE cnpj = '$termo' OR nome LIKE '%$termo%' ";
+            $sql .= "WHERE cnpj = '$termo' OR nome LIKE '%($termo)%' ";
```

Comment:
nao existe o parenteses no LIKE, somente no IN

Reply:
Resolvido!!